src/main/java/com/skankhunt220/service/CityService.java

```diff
@@ -0,0 +1,53 @@
+package com.skankhunt220.service;
+
+import java.util.Optional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+import com.skankhunt220.entity.City;
+import com.skankhunt220.repository.city.CityRepository;
+
+@Service
+public class CityService {	
+	@Autowired
+	private CityRepository cityRepository;
```

Comment:
instead of @Autowired use `final static` and annotation from lombock @REquired ArgConstructor